inventory/sample/group_vars/all/all.yml

```diff
@@ -88,6 +88,10 @@ no_proxy_exclude_workers: false
 ## Set to true to allow pre-checks to fail and continue deployment
 # ignore_assert_errors: false
 
+## Set to true to skip 'Stop if even number of etcd hosts' assert check
+## Running an even number of etcd nodes is *not* recommended for production clusters
+skip_etcd_even_nodes_check: false
+
```

Comment:
Remove this. The inventory sample is for "typical" settings, and not only this should not be a common case, this should never be in the inventory, probably only specified in extra_var.